ecmd-core/capi/ecmdUtils.C

```diff
@@ -942,7 +942,7 @@ uint32_t ecmdDisplayScomData(ecmdChipTarget & i_target, ecmdScomData & i_scomDat
 }
 uint32_t ecmdDisplayScomData(ecmdChipTarget & i_target, ecmdScomDataHidden & i_scomData, ecmdDataBuffer & i_data, const char* i_format, std::string *o_strData) {
```

Comment:
In this branch there's another ecmdDisplayScomData method above that needs the same changes.